sde_sampler/losses/oc.py

```diff
@@ -369,3 +373,124 @@ def eval(
         return BaseOCLoss.compute_results(
             rnd, compute_weights=compute_weights, ts=ts, samples=samples, xs=xs
         )
+
+
+class ExponentialIntegratorSDELoss(BaseOCLoss):
+    def __init__(self, *args, **kwargs):
+        # Only loss.method = KL implemented
+        super().__init__(*args, **kwargs)
+        self.alpha = kwargs.get("alpha", 1)
+        self.sigma = kwargs.get("sigma", 1)
+
+    def simulate(
+        self,
+        ts: torch.Tensor,
+        x: torch.Tensor,
+        terminal_unnorm_log_prob: Callable,
+        reference_log_prob: Callable | None = None,  
+        compute_ito_int: bool = False,
+        change_sde_ctrl: bool = False,
```

Comment:
Can we remove `change_sde_ctrl` (here and in the functions calling it) since it is not implemented?